src/hpr_config.erl

```diff
@@ -0,0 +1,178 @@
+-module(hpr_config).
+
+-export([
+    init/0,
+    stop/0,
+    update_routes/1,
+    insert_route/1,
+    lookup_devaddr/1,
+    lookup_eui/2
+]).
+
+-define(DEVADDRS_ETS, hpr_config_routes_by_devaddr).
```

Comment:
Since the keys are different for dev addr entries and eui entries, do we need two ETS tables? The shape of the key distinguishes them. Or maybe define a record for the key?

A comment describing the row format of the tables would be helpful.

Reply:
I was thinking of performance lookups wouldn't it be better with 2 tables?

Reply:
`ets` is pretty fast and we've set the `read_conucrrency` option. Having two tables makes the insert/update code a little more complicated because we have to separate out the two types of values. On the other hand, we want to optimize reads since the updates aren't expected to be that frequent(?).

Reply:
`ets` is pretty fast and we've set the `read_conucrrency` option. Having two tables makes the insert/update code a little more complicated because we have to separate out the two types of values. On the other hand, we want to optimize reads since the updates aren't expected to be that frequent(?).